lib/publify/api.rb

```diff
@@ -0,0 +1,14 @@
+require 'net/http'
+
+module Publify
+  class API
+    def self.latest_links(limit)
+      uri = URI("http://#{PUBLIFY_HOSTNAME}/articles.json")
+      response = Net::HTTP.get uri
```

Comment:
Yep, could we make sure this isn't unbounded. Also, how is this being called? Is it frequent, is this response cached?